test/python/WMCore_t/WMSpec_t/samples/MultiMergeProductionWorkload.py

```diff
@@ -16,7 +16,7 @@
 #  //
 # // Set up the basic workload task and step structure
 #//
-workload = newWorkload("MultiProduction")
+workload = newWorkload("MultiOutputs")
```

Comment:
Yes, this sample is still used by https://github.com/dmwm/WMCore/blob/master/test/python/WMCore_t/WorkQueue_t/Policy_t/Start_t/MonteCarlo_t.py